haproxy/templates/secret-check-auth-redisk8s.yaml

```
@@ -0,0 +1,19 @@
+{{- if and (eq .Values.haproxy.config "redisk8s") .Values.haproxy.redisk8s.check.enabled  }}
+{{- if (empty .Values.haproxy.redisk8s.check.existingSecret) }}
```

```suggestion
{{- if (empty .Values.haproxy.check.existingSecret) }}
```

If I'm not mistaken?